tools/recalc.py

```diff
@@ -212,9 +212,23 @@ async def main(argv: Optional[Sequence[str]] = None) -> int:
     if len(argv) == 0:
         argv = ["--help"]
 
-    parser = argparse.ArgumentParser(description="Recalculate performance for scores")
+    parser = argparse.ArgumentParser(
+        description="Recalculate performance for scores and/or stats",
+    )
 
     parser.add_argument("-d", "--debug", action="store_true")
+    parser.add_argument(
+        "--scores",
+        description="Recalculate scores",
+        action="store_true",
+        default=True,
+    )
+    parser.add_argument(
+        "--stats",
+        description="Recalculate stats",
+        action="store_true",
+        default=True,
+    )
```

Comment:
In hindsight.. is it even possible to disable this now? I think with `store_true` this is strictly only able to be set to True..

Perhaps we should flip these to `--no-stats-recalc` `--no-scores-recalc` with a default of False

Reply:
lol stupid oversight on my end i think
i hate argparse